### PR TITLE
Move action buttons to controls area of instance overview

### PR DESCRIPTION
**Pull request by lorumic**

### CODE REVIEW

File: static/js/InstanceOverview.tsx [hunk 1]
```diff
@@ -28,6 +34,47 @@ const InstanceOverview: FC<Props> = ({ instanceName }) => {
     queryFn: () => fetchInstance(instanceName),
   });
 
+  useEffect(() => {
+    if (instance) {
+      setControls(
+        <div style={{ display: "inline", marginRight: "1rem" }}>
```
Comment (edlerd) [inline static/js/InstanceOverview.tsx:40]: We should avoid adding CSS in JS. Better fix it with a vanilla class.

I think the problem is the tooltip in the delete button. So you could give the tooltip the right utility class from vanilla, to get correct spacing.

Comment (edlerd) [reply static/js/InstanceOverview.tsx:40]: PS: Not sure vanilla has utility classes for horizontal spacing. Maybe ask the vanilla pros for advice.

Comment (anthonydillon) [reply static/js/InstanceOverview.tsx:40]: No, this is generally handled by the grid.

File: static/js/InstanceOverview.tsx [hunk 2]
```diff
@@ -28,6 +34,47 @@ const InstanceOverview: FC<Props> = ({ instanceName }) => {
     queryFn: () => fetchInstance(instanceName),
   });
 
+  useEffect(() => {
+    if (instance) {
+      setControls(
+        <div className="u-instance-overview-controls">
```
Comment (edlerd) [inline static/js/InstanceOverview.tsx:40]: This is inline by default. So we can use an empty fragment here as a wrapper. And clean-up the `u-instance-overview-controls` class.

**Review by edlerd — APPROVED** [review]

File: static/js/InstanceOverview.tsx (hunk 2 above)
Comment (lorumic) [reply static/js/InstanceOverview.tsx]: Very strange. I tried that approach before first, and it wasn't working. It was causing buttons to be stacked vertically. That's why I had applied that style. Anyway, now that I tried again, it works as you said. Done @ [1ea1594](https://github.com/canonical/lxd-ui/commit/1ea159429bc0928d054872d87ec3c930dbe025d7)

**Pull request merged**